### DISCUSSION TITLE
Resources for Learning the Basics of App Development (Connections, Storage, etc.)

### DISCUSSION THREAD

**Discussion by LimaVsC** [zauberzeug/nicegui#4133] [Q&A]
### Question

I'm at the point where I’m considering making my app available to others, but since this is my first time developing an app, I’m unsure how to handle client connections, storage, and similar aspects.

Can someone recommend some reading resources or guides on the basics I need to learn to accomplish this?

Thank you in advance for your help!

**Comment by rodja** [comment]
Great to hear @LimaVsC! Maybe https://github.com/zauberzeug/nicegui/wiki/fly.io-Deployment will help you with launching your app.
